src/pre-core-exemptions/README.md

```diff
@@ -0,0 +1,53 @@
+# Pre Core Exemptions
+
+This package serves as a way for users deploying uds-core to pass exemptions to core for things that are deployed before core.
```

Comment:
```suggestion
This package serves as a way for users deploying uds-core to create exemption resources for things that are deployed before core.
```
Maybe? Technically is a "pass through" but I think its more clear to just say this is a way to create exemptions? No strong opinion here.